community/deb.sh

```diff
@@ -79,10 +79,19 @@ detect_os ()
   echo "Detected operating system as $os/$dist."
 }
 
+add_pgdg ()
+{
+  echo "Registering PostgreSQL PGDG dependencies..."
+  echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+  apt-get install wget ca-certificates
+  wget --quiet --no-check-certificate -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
```

Comment:
Eeeehhh, a dependency on _both_ `curl` _and_ `wget`? We should change this to `curl`.